examples/analog_hamiltonian_simulation/09_Noisy_quantum_evolution_of_Bell_states.ipynb

```diff
@@ -0,0 +1,1628 @@
+{
```

Comment:
<p>Let's emphasize the logic here: Each shot is simulated with a different sample of the random variables. (While it's expensive, there is a beauty to it: this is how our world functions, each shot is experiences a different instance of the uncontrolled environmental factors.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>